src/qt/font.cpp

```diff
@@ -313,13 +335,44 @@ void wxFont::SetUnderlined( bool underlined )
     M_FONTDATA.SetUnderlined(underlined);
 }
 
+void wxFont::SetStrikethrough(bool strikethrough)
+{
+    AllocExclusive();
+
+    M_FONTDATA.SetStrikethrough(strikethrough);
+}
+
 void wxFont::SetEncoding(wxFontEncoding encoding)
 {
     AllocExclusive();
 
     M_FONTDATA.SetEncoding(encoding);
 }
 
+bool wxFont::SetNativeFontInfo(const wxString& info)
```

Comment:
The implementation seems exactly identical to the base class version, why do we need it?

Reply:
We do not, I will remove it!